lighthouse-core/gather/connections/connection.js

```diff
@@ -46,15 +46,16 @@ class Connection {
    * Call protocol methods
    * @param {!string} method
    * @param {!Object} params
+   * @param {boolean=} silent
```

Comment:
open to much better names, `silenceError` felt like it would supress the error itself, `silenceErrorLogging` too verbose?